syncer/syncer.go

```diff
@@ -263,6 +263,7 @@ func NewSyncer(cfg *config.SubTaskConfig) *Syncer {
 		syncer.ddlExecInfo = NewDDLExecInfo()
 	}
 
+	syncer.done = nil
```

Comment:
we can initialize `syncer.done` at L#229

Reply:
oh i see what you mean, fixed at https://github.com/pingcap/dm/pull/226/files#diff-ea4abf20044394891136e70431919a4fR229